specs/latest/1.0/webgl.idl

```diff
@@ -756,7 +763,7 @@ WebGLRenderingContext includes WebGLRenderingContextOverloads;
 
 [Exposed=(Window,Worker),
  Constructor(DOMString type,
- optional WebGLContextEventInit eventInit)]
+ optional WebGLContextEventInit eventInit = {})]
```

Comment:
maybe I'm missing something, but why this change in this PR?

Reply:
No idea! :) Will revert.